src/Runner.Listener/JobDispatcher.cs

```diff
@@ -952,8 +951,10 @@ private async Task LogWorkerProcessUnhandledException(Pipelines.AgentJobRequestM
                 ArgUtil.NotNull(timeline, nameof(timeline));
                 TimelineRecord jobRecord = timeline.Records.FirstOrDefault(x => x.Id == message.JobId && x.RecordType == "Job");
                 ArgUtil.NotNull(jobRecord, nameof(jobRecord));
+                var unhandledExceptionIssue = new Issue() { Type = IssueType.Error, Message = errorMessage };
+                unhandledExceptionIssue.Data[Constants.Runner.WorkerCrashIssueDataKey] = string.Empty;
```

Comment:
service will check the issue data to decide whether fire product trace event.

Reply:
Think about:

Should we make this more generic? Do we have any other cases that warrant a more generic telemetry mechanism, e.g. `[...].Data["_telemetry"] = WORKER_CRASH;<MESSAGE>`

^ That way we can start adding TYPE;MESSAGE more generically to issues?